include/bsp/iotc_bsp_time.h

```diff
@@ -61,6 +61,12 @@ iotc_time_t iotc_bsp_time_getcurrenttime_seconds();
  */
 iotc_time_t iotc_bsp_time_getcurrenttime_milliseconds();
 
+/**
+ * @function
+ * @brief Returns current monotonic time in microseconds.
```

Comment:
I think the term "current" adds confusion.
What about `@brief Returns monotonically increasing time in microsecond precision.` ?